### PR TITLE
[CBRD-25043] refactor functions related to isspace, tolower, and toupper

### PR DESCRIPTION
**Pull request by ctshim**
http://jira.cubrid.org/browse/CBRD-25043

* Refactor the functions in the following list to inline.
  -  char_islower, char_isupper, char_isalpha, char_isdigit, char_isalnum, 
  -  char_isspace, char_iseol, char_isxdigit, char_tolower, char_toupper, 
  - char_isupper_iso8859, char_islower_iso8859, char_tolower_iso8859, char_toupper_iso8859
*  Each function can immediately check and obtain values from a pre-prepared array.

### CODE REVIEW

File: src/base/chartype.c [hunk 1]
```diff
@@ -27,188 +27,240 @@
...
+      str[len] = '\0';
+    }
+
+  return (str);
 }
 
-/* Specialized for ISO 8859-1 */
+//=============================================================================
+#if 0
+// build mapper and prop table
```
Comment (beyondykk9) [inline src/base/chartype.c:203]: "#if 0" is intensive for the future?

File: src/base/intl_support.c [hunk 2]
```diff
@@ -66,9 +66,8 @@
               && ((unsigned char) ch <= (unsigned char) 0xfe) )
 #endif
 
-#define CHAR_BYTE_TO_LOWER(c) ((c) + ('a' - 'A'))
-
-#define CHAR_BYTE_TO_UPPER(c) ((c) - ('a' - 'A'))
+//#define CHAR_BYTE_TO_LOWER(c) ((c) + ('a' - 'A'))
```
Comment (beyondykk9) [inline src/base/intl_support.c]: remove the codes if it's not necessary.

File: src/broker/broker_util.c [hunk 3]
```diff
@@ -102,7 +102,7 @@ ut_access_log (int as_index, struct timeval *start_time, char error_flag, int er
     clt_appl = (char *) "-";
   for (p = clt_appl; *p; p++)
     {
-      if (*p == ' ' || *p == '\t' || *p == '\n' || *p == '\r')
+      if (char_isspace2 (*p))
```
Comment (beyondykk9) [inline src/broker/broker_util.c:105]: why not char_isspace ?

**Review by hyunikn — APPROVED** [review]

**Review by hgryoo — APPROVED** [review]

**Review by kisoo-han — APPROVED** [review]

File: src/base/intl_support.c (hunk 2 above)
Comment (ctshim) [reply src/base/intl_support.c]: OK, Thank you.

File: src/broker/broker_util.c (hunk 3 above)
Comment (ctshim) [reply src/broker/broker_util.c:105]: I wrote a brief description in chartype.h.

```
/* In some codes, only four items (' ', '\t', '\n', '\r') were checked.
 * In order to maintain its original form, it was given a different name. */
#define char_isspace2  char_isspace
```

File: src/base/chartype.c (hunk 1 above)
Comment (ctshim) [reply src/base/chartype.c:203]: This content has nothing to do with actual practice.
However, this is code to create contents such as char_upper_mapper, char_properties, etc.
This is for reference when there is a need to change the structure of the content in the future.

I will reinforce the comment.

**Review by beyondykk9 — APPROVED** [review]

**Pull request merged**